packages/fluentui/react-northstar/src/components/Tree/TreeTitle.tsx

```diff
@@ -116,6 +154,33 @@ const TreeTitle: React.FC<WithAsProp<TreeTitleProps>> & FluentComponentStaticPro
       })}
     >
       {childrenExist(children) ? children : content}
+      {// if tree item has subtree but it is not selectable explicitly, do not redner any selection indicator
+
+      selectable &&
```

Comment:
maybe just use a variable for all this condition. then use that variable in the JSX

Reply:
please review again, changes were applied by Charles :)